canonical/canonical.com
# Documentation practice

## Done

Based on @mtruj013's [pr](https://github.com/canonical/canonical.com/pull/905). I changed a few things so pushing to a separate pr so as to not break anything.

This is a bit different from the design, as I talked to Daniele since we did the pages and he expressed a preference for a denser treatment on documentation pages, and liked placing headings on the side and clamping the heading hierarchy to h4. 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check tab functionality on all 3 tabs

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3737
